frontend/tests/utils/helpers.ts

```diff
@@ -64,6 +64,7 @@ export async function goToPage(
   page: Page
 ): Promise<void> {
   await page.goto(url, { timeout: GO_TO_PAGE_TIMEOUT_MS });
+  await page.waitForLoadState("networkidle");
```

Comment:
to avoid flakiness, wait for network idle for all `goTo()` calls, so data will be loaded before we start test steps